Scripts/interactable/Locomotion/SmartControl.lua

```diff
@@ -136,4 +169,17 @@ function SmartControl.server_onFixedUpdate(self, dt)
 			sm.joint.setTargetLength( v, 0, pistonspeed, maxImpulse )
 		end
 	end
+
+	if self.sv_saved_fuel_points ~= self.sv_fuel_points then
+		self.sv_saved_fuel_points = self.sv_fuel_points
+		self.storage:save(self.sv_fuel_points)
```

Comment:
Same thing here, this calls `self.storage:save` every tick.

Reply:
Fixed